src/data/stages.ts

```diff
@@ -18,6 +18,9 @@ export const bridgedChains = {
   story: {
     generalDisplayName: "Story",
   },
+  hyperliquidEVM: {
```

Comment:
```suggestion
  hyperliquid: {
```

So that we have nice links

Reply:
https://github.com/humanode-network/biomapper-docs/pull/62/commits/0ef7ce76bb1e0a44d7e672adbc779e4a710a3f56